bi-connectors/TableauConnector/README.md

```diff
@@ -0,0 +1,29 @@
+## Connector Download
+
+The most recent build could be donwloaded from an automated CI workflow: [link](https://github.com/opensearch-project/sql/actions/workflows/bi-connectors.yml).
```

Comment:
Recommend changing from: 
`The most recent build could be donwloaded from an automated CI workflow`
to:
`The Tableau connector is available to be downloaded from the automated CI workflow`